scripts/release_notes/common.py

```diff
@@ -253,7 +253,7 @@ def github_data(pr_number):
     )
     query = run_query(query)
     if query.get("errors"):
-        raise Exception(query["errors"])
```

Comment:
I think we want to be loud about the number of errors that occur. Since if 1 or 2 commits are busted that is probably fine but could this potential hide some problems?

Reply:
I see, makes sense, I can add some handling for that